.github/workflows/buildtest.yml

```diff
@@ -64,3 +64,31 @@ jobs:
           echo ""
           exit 1
         fi
+
+  validate-webhooks:
```

Comment:
I added this job to validate the optimized webhooks JSON is equivalent to the full webhooks JSON file.

Refer changes to the file eventPayloads.test.ts in this PR